test/setup.sh

```diff
@@ -4,4 +4,20 @@ set -eu -o pipefail
 TTY=
 if [ -t 1 ]; then TTY=-t; fi
 
-docker exec -i $TTY test-destination_ubuntu-1 /work/test/dockerfiles/install-ubuntu.sh
+docker exec -i $TTY \
+  -e DESTINATION_NAME=ubuntu \
+  -e DESTINATION_ADDR=127.0.0.1:8220 \
+  -e INFRA_ACCESS_KEY=dest000001.ubuntuubuntuubuntuubuntu \
+  test-destination_ubuntu-1 /work/test/dockerfiles/install-debian.sh
+
+docker exec -i $TTY \
+  -e DESTINATION_NAME=debian \
+  -e DESTINATION_ADDR=127.0.0.1:8221 \
+  -e INFRA_ACCESS_KEY=dest000002.debiandebiandebiandebian \
+  test-destination_debian-1 /work/test/dockerfiles/install-debian.sh
+
+docker exec -i $TTY \
+  -e DESTINATION_NAME=redhat \
+  -e DESTINATION_ADDR=127.0.0.1:8222 \
+  -e INFRA_ACCESS_KEY=dest000003.redhatredhatredhatredhat \
+  test-destination_redhat-1 /work/test/dockerfiles/install-redhat.sh
```

Comment:
We could do these setup steps in parallel, but that ends up making the logging a bit harder to consume (when it comes time to debug a failure). We'd have to send the output to a file and save the file as an artifact.

I think we'll want to do that when we had the 4th or 5th destination, but for now it seems to run just fast enough we can do it sequentially.